src/pages/components/AllCompensations/components/AllCompensationsTable/AllCompensationsTable.tsx

```diff
@@ -9,12 +9,13 @@ import { convertAmountsToNominals } from '../../convertAmountsToNominals'
 
 export const AllCompensationsTable = observer(({
```

Comment:
This became a big component that requires decomposition on a subset of components. Let's plan it as a separate activity, not in the scope of this PR.